### ISSUE TITLE
Discriminator loss (D_real & D_fake loss) is always between 0.001 to 1.5

### ISSUE THREAD

**Issue by kalai2033** [junyanz/pytorch-CycleGAN-and-pix2pix#802]
Hi,
    I training a pix2pix model with my own dataset with 8000 images. The loss for G_L1 converges very fast like in 20 epochs or so. But the other losses like D_real and D_fake always remains between 0.0001 to 1.5 in almost all 64 epochs i have trained so far. Is my model is learning or not? I have attached the logs as txt file.  
[logloss.txt](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/files/3738812/logloss.txt)

**Comment by kalai2033** [comment]
@junyanz Thanks for your info. It is agreeable for losses to oscillate. But my doubt is discriminator loss always between 0.001 to 1. I trained an model just with 100 images for 200 epochs. The result seems to be decent for 100 odd images. I have attached the screenshots logloss plot for reference. Is it normal behaviour for discrimator loss to be between 0 and 1?

![Screenshot from 2019-10-17 16-04-21](https://user-images.githubusercontent.com/49688647/67046111-e8066180-f12f-11e9-90ca-f0642d266021.png)
![Screenshot from 2019-10-17 16-04-41](https://user-images.githubusercontent.com/49688647/67046112-e8066180-f12f-11e9-9365-0ddcec9da4cb.png)
![Screenshot from 2019-10-17 16-04-26](https://user-images.githubusercontent.com/49688647/67046113-e8066180-f12f-11e9-908c-a35f1791348e.png)
